### PR TITLE
Adding the fix to not remove tags when performing ssl bind operation

### LINKED ISSUE #10881: "az webapp config ssl bind" wipes tags
> ### `az feedback` auto-generates most of the information requested below, as of CLI version 2.0.62

**Describe the bug**
When "az webapp config ssl bind" command is run (including within the Azure Cloud Shell) the tags that were applied to the App Service are removed.

**To Reproduce**
- Apply tags to the App Service
- Run the "az webapp config ssl bind" command to bind a TLS certificate to the App Service
- Check the tags on the App Service

**Expected behavior**
The tags of the App Service should not be removed.  The tags should be maintained throughout this operation.

**Environment summary**
Cloud Shell (bash)

**Additional context**
We have tag policies applied in our environment.  Running this command and wiping the tags causes the resource to become non-compliant, preventing the command from being run.

### PR DESCRIPTION
**Pull request by panchagnula**
Fixes #10881
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/appservice/custom.py [hunk 1]
```diff
@@ -205,10 +205,14 @@ def update_app_settings(cmd, resource_group_name, name, settings=None, slot=None
         for s in src:
             try:
                 temp = shell_safe_json_parse(s)
+                print(temp)
                 if isinstance(temp, list):  # a bit messy, but we'd like accept the output of the "list" command
                     for t in temp:
+                        print(t)
                         if t.get('slotSetting', True):
+                            print("Running with slot settings")
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/appservice/custom.py]: Too many print(). Use logger?

Comment (panchagnula) [reply src/azure-cli/azure/cli/command_modules/appservice/custom.py]: Sorry that was not meant to be here. I will remove.

**Comment by yonzhan** [comment]
@qwordy please help with this PR.

**Comment by anarchitect** [comment]
May I know when this fix will be effective on these Microsoft Hosted Agents?  I am running an inline Azure CLI task in Azure Pipelines.  Thanks.

**Comment by yonzhan** [comment]
After next Azure CLI release. You can see all release milestones in GitHub.

**Comment by panchagnula** [comment]
This is marked as for S160 release see the MileStone here https://github.com/Azure/azure-cli/milestones

**Comment by qwordy** [comment]
You see the change in Azure CLI 2.0.76. Official Release: 11/04/2019.

**Pull request merged**